initrd/init

```diff
@@ -16,50 +16,36 @@ mkdir /proc /sys /dev /tmp /boot /media 2>&- 1>&-
 mount /dev 2>/dev/ttyprintk
 mount /proc 2>/dev/ttyprintk
 mount /sys 2>/dev/ttyprintk
-mount /sys/firmware/efi/efivars
-
-# Setup the pty psudeo filesystem
-mkdir /dev/pts
```

Comment:
What's wrong with always mounting `/dev/pts`?

Reply:
Mmm... `CONFIG_UNIX98_PTYS=y` isn't always built in the kernel so mounting devpts can fail but I guess that's not awful (added `CONFIG_LINUX_PTY` in board config in PR #331).